includes/wikia/DefaultSettings.php

```diff
@@ -1676,6 +1676,11 @@
  */
 $wgDisableWAMOnHubs = false;
 
+/* @name wgTreatAsCorporate
+ * Treat a wiki as if it was a corporate wiki
+ */
+$wgTreatAsCorporate = false;
```

Comment:
Isn't what I did here? :neutral_face:
> Separate variable and logic completely separated from existing variables.

This separates logic of 'adding to the corporate dropdown' from 'having a Wikia Stats instead of WAM'. The new variable is checked only while including any other wiki than the ones with `$wgEnableWikiaStatsExt` in the *Corporate Footer* language chooser dropdown—the old variable is removed from this check. Imagine it's renamed from `$wgTreatAsCorporate` to `$wgDisplayInCorporateFooterDropdown` (or something similar). Are you following? Am I following you correctly? :sweat_smile: